.github/workflows/docs-teardown.yml

```diff
@@ -7,6 +7,7 @@ on:
       - "dev"
       - "5.x"
       - "4.[0-9]"
+      - "gql-notifications"
```

Comment:
Same as for `docs-pr-checks.yml`. Assuming we are going to delete this branch after merging, we don't need to include it here